docs/src/content/docs/commands/EXPIRE.md

```diff
@@ -104,3 +104,6 @@ This example shows what happens when trying to set an expiration on a non-existe
 ## Alternatives
 
 - Use `EXPIREAT` command for more precise expiration control based on Unix timestamps
+
+## Notes
+- Please refer to the [TTL](/commands/ttl), [PERSIST](/commands/persist) and [EXPIREAT](/commands/expireat)  commands documentation for more information.
```

Comment:
same as above. Lets add the infomation in Alternatives section. Also lets keep the style as [`TTL`](...URL..)